template/Goerli-DssSpell.sol

```diff
@@ -40,17 +42,20 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //    https://ipfs.io/ipfs/QmPgPVrVxDCGyNR5rGp9JC5AUxppLzUAqvncRJDcxQnX1u
```

Comment:
did you use a script to generate this ?

Reply:
Nope, the IPFS hash was taken from `spells-mainnet`

Reply:
But now that I think more on it, it might be nice to add a script to generate rates based on the commented formula to ease the spell crafting and reviewing process and eventually automate IPFS Rates Table updates (that might need some further updates, like including 0.01%).